packages/core/src/api/cardano/CardanoSignTransaction.ts

```
@@ -48,6 +53,16 @@ export default class CardanoSignTransaction extends BaseMethod<any> {
 
     const { payload } = this;
 
+    // convert legacy parameters to new parameter
+    // payload.auxiliaryData.governanceRegistrationParameters are legacy params kept for backward compatibility (for now)
+    if (payload.auxiliaryData && payload.auxiliaryData.governanceRegistrationParameters) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)